php/post-types/class-block-post.php

```diff
@@ -325,7 +325,8 @@ public function add_meta_boxes() {
 		);
 
 		if ( isset( $post->post_name ) && ! empty( $post->post_name ) ) {
-			$template = block_lab_locate_template( 'blocks/block-' . $post->post_name . '.php', '', true );
+			$locations = block_lab_template_locations( $post->post_name, 'block' );
```

Comment:
What do you think about not passing the second argument of `'block'`? It looks to be the default argument.

Reply:
Fixed in e970caa.

Reply:
Thanks, that looks good!